ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/WebSocketTest.kt

```diff
@@ -271,4 +272,38 @@ class WebSocketTest : ClientLoader() {
             assertNotNull(session.closeReason.await())
         }
     }
+
+    @Test
+    fun testWebsocketRequiringSubProtocolWithSubProtocol() = clientTests(ENGINES_WITHOUT_WS) {
+        config {
+            install(WebSockets)
+        }
+
+        test { client ->
+            client.webSocket(
+                "$TEST_WEBSOCKET_SERVER/websockets/sub-protocol",
+                request = {
+                    header(HttpHeaders.SecWebSocketProtocol, "test-protocol")
+                }
+            ) {
+                send(Frame.Text("test"))
+                val result = incoming.receive() as Frame.Text
+                assertEquals("test", result.readText())
+            }
+        }
+    }
+    @Test
```

Comment:
codestyle will fail here, because it requires a new line in between two methods

Reply:
This should be fixed now.